INSTALL.rst

```diff
@@ -1,123 +1,104 @@
 
-Now, clone scikit-image on your local computer, and install::
 
-  git clone https://github.com/scikit-image/scikit-image.git
-  cd scikit-image
-  pip install -e .
+Should you need to have access to the wheels offline, they can be downloaded
+manually from `PyPi <https://pypi.org/project/scikit-image/>`__.
 
-To update the installation::
+conda
+=====
 
-  git pull  # Grab latest source
-  pip install -e .  # Reinstall
+When using conda, you may find the following bash commands useful::
 
-Platform-specific notes follow below.
+  # Create a conda environment named ``skimage``
+  conda create --name skimage
+  # Activate it
+  conda activate skimage
+  conda install scikit-image
 
-a. Windows
-``````````
-If you experience the error ``Error:unable to find vcvarsall.bat`` it means
-that your computer does not have recommended compilers for Python. You can
-either download and install Windows compilers from `here`_  or use
-`MinGW compilers`_ . If using `MinGW`, make sure to correctly configure
-``distutils`` by modifying (or create, if not existing) the configuration file
-``distutils.cfg`` (located for example at
-``C:\Python26\Lib\distutils\distutils.cfg``) to contain::
+Now, whenver you want to use this specific version of scikit-image, make sure
+you activate your conda environment by typing::
 
-  [build]
-   compiler=mingw32
+  # Activate the specific conda environment
+  conda activate skimage
+  # Use python as you wish
 
-A run-through of the compilation process for Windows is included in
-our `setup of Azure Pipelines`_ (a continuous integration service).
+While the default Anaconda installation provides scikit-image, you may wish to
+explore `conda-forge <https://conda-forge.org/>`__ for a more updated version.
+If you are new to conda, you may also find the following
+`introduction to conda <https://kaust-vislab.github.io/introduction-to-conda-for-data-scientists/>`__
+useful.
 
-.. _miniconda: http://conda.pydata.org/miniconda.html
-.. _python.org: http://python.org/
-.. _setup of Azure Pipelines: https://github.com/scikit-image/scikit-image/blob/master/azure-pipelines.yml
-.. _here: https://wiki.python.org/moin/WindowsCompilers#Microsoft_Visual_C.2B-.2B-_14.0_standalone:_Visual_C.2B-.2B-_Build_Tools_2015_.28x86.2C_x64.2C_ARM.29
-.. _venv: https://docs.python.org/3/library/venv.html
-.. _virtual environments: https://docs.python-guide.org/en/latest/dev/virtualenvs/
-.. _MinGW compilers: http://www.mingw.org/wiki/howto_install_the_mingw_gcc_compiler_suite
+3. Source based installation for development purposes
+-----------------------------------------------------
 
-b. Debian and Ubuntu
-````````````````````
-Install all the required dependencies::
+Development installation instructions can be found in the Contribution guide.
+TODO: provide a cross link to this????
 
-  sudo apt-get install python3-matplotlib python3-numpy python3-pil python3-scipy python3-tk
+4. Using the operating system's package manager
```

Comment:
I think I do not understand this title as you do. I would expect to read something like "use apt install python3-skimage" [that I would recommend], but you rather seem to think about "sudo pip install ..." [I discourage as well].

Should we entitle: "Using the operating system's pip command"

Reply:
Yes, using "sudo apt" is what I mean by operating system's package manager. 

My feeling is that it should not be used as it really installs out of date versions of packages.

For example, 19.10 still installs 0.14.2 for python 3.

Reply:
@sciunto @hmaarrfk you are both right. Since we have tons of *nix flavors, each one with their different package system (`deb`, `rpm`, `pkg`, `orb`, `apk`, `pup`, sources, not mentioning snaps and flatpaks and whatnot... geez), I would think on removing this section and supporting `pip` and `conda-forge` packages currently. What do y'all think?

Reply:
I think people want to know how to do it. We can discourage it, but if we don't document it, I think people will get confused by the omission.